specs/triggers/tutorial.spec

```diff
@@ -4,7 +4,7 @@ PIPELINES-06
 Pre condition:
   * Validate Operator should be installed
 
-## Run Triggers tutorial (by Automatically configuring users webhook to git repo): PIPELINES-06-TC01
+## Run Triggers tutorial (by Automatically configuring users webhook to git repo): PIPELINES-07-TC01
```

Comment:
Don't change the existing test cases. Let's keep this test case as `PIPELINES-06-TC01`